pkg/server/admin_test.go

```diff
@@ -400,6 +400,12 @@ func TestAdminAPITableDetails(t *testing.T) {
 	testAdminAPITableDetailsInner(t, "test", "tbl")
 }
 
+// #14056
```

Comment:
These lint rules don't apply to test functions.

If you are feeling altruistic, you could convert this test and its friends to sub-tests:

```golang
func TestAdminAPITableDetails(t *testing.T) {
  defer leaktest.AfterTest(t)()

  for _, tc := range []struct{
    name, dbName, tableName string
  }{
   { name: "lower", dbName: "test", tableName: "tbl" },
   { name: "lower with space", dbName: "test test", tableName: "tbl tbl" },
   { name: "upper", dbName: "TEST", tableName: "TBL" },
  }{
    t.Run(tc.name, func(t *testing.T) {
      // body of testAdminAPITableDetailsInner
    })
  }
}
```